src/compiler/scala/tools/nsc/backend/jvm/BCodeHelpers.scala

```diff
@@ -1161,7 +1170,12 @@ object BCodeHelpers {
   val ExcludedForwarderFlags = {
     import scala.tools.nsc.symtab.Flags._
     // Should include DEFERRED but this breaks findMember.
-    SPECIALIZED | LIFTED | PROTECTED | STATIC | EXPANDEDNAME | BridgeAndPrivateFlags | MACRO
+    // Note that BRIDGE is *not* excluded. Trying to exclude bridges by flag doesn't work, findMembers
+    // will then include the member from the parent (which the bridge overrides / implements).
```

Comment:
This comment is true. Unfortunately there was an unintended side-effect: before this PR, the `findMembers` call for forwarder methods would return the member from the parent overriddent by the bridge. This member is potentially abstract, in which case the forwarder is not emitted (https://github.com/scala/scala/blob/v2.12.7/src/compiler/scala/tools/nsc/backend/jvm/BCodeHelpers.scala#L892).

After this PR, `findMembers` returns the bridge, which is concrete, so a forwarder is added. This is the cause for https://github.com/scala/bug/issues/11207.